### ISSUE TITLE
two player play at the same time

### ISSUE THREAD

**Issue by xzjs** [ZeBobo5/Vlc.DotNet#690]
I have an issue / a question (pick one) about Vlc.DotNet.

# Generic information

- **Vlc.DotNet version** : (3.1.0)
- **Vlc.DotNet project used** : (WinForms)
- **libvlc version** : (3.0.14） + nutget
- **.net version** : (5.0)
- **Project language** : (C#)
- **Project build architecture** : AnyCPU
- **Operating system** : Windows x64

# Summary

I have two players. I want them to play at the same time, what should I do?

**Comment by jeremyVignelles** [comment]
Use two media player controls?

Also, please consider switching to LibVLCSharp, this project is deprecated.

**Comment by xzjs** [comment]
yes, but i have done a lot of work in Vlc.DotNet ,are there any solutions？

**Comment by mfkl** [comment]
> Use two media player controls?

**Comment by xzjs** [comment]
> Use two media player controls?

yeah,i use two media player controls,and i play meida by this code
```
var actions = new Action[] {
    ()=>vlcControl1.Play(),
    ()=>vlcControl2.Play()
};
Parallel.Invoke(actions);
```
The two media are not playing simultaneously

**Comment by mfkl** [comment]
there is no easy way to sync 2 mediaplayers.

**Comment by ZeBobo5** [comment]
You can try playing 2 videos in one media player like https://www.vlchelp.com/how-to-open-two-videos-side-by-side/
Not tested
